modules/edu.gemini.seqexec.model/src/test/scala/edu/gemini/seqexec/model/SharedModelArbitraries.scala

```diff
@@ -26,6 +26,12 @@ object SharedModelArbitraries {
     } yield SequencesQueue(Conditions.default, Some(Operator("operator")), b)
   }
 
+  implicit val clientIdArb: Arbitrary[ClientID] = Arbitrary {
+    for {
+      i <- Gen.uuid
+    } yield i
+  }
+
```

Comment:
Can't you just do:
`implicit val clientIdArb: Arbitrary[ClientID] = Arbitrary(Gen.uuid)`
?

Reply:
No reason, I'll change it as you suggest